.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+# Purpose
+
+**Type of PR**
+* [ ] Bug Fix
+* [ ] Feature Work
+* [ ] Revert
+* [ ] Refactor / Code Cleanup
+* [ ] Other (fill in...)
+
+**Review Urgency**
```

Comment:
Maybe remove this, just cause I think it's an odd piece of information for developers to be filling out of they are open source devs